package.json

```diff
@@ -82,6 +100,8 @@
     "@types/react-modal": "^3.0.0",
     "@types/react-notify-toast": "^0.5.0",
     "@types/simple-oauth2": "^4.0.0",
+    "@types/slate-html-serializer": "^0.6.4",
```

Comment:
Do you need these types? Doesn't look like you import `slate-html-serializer` and the text plugin provides its own types.